doc/nrf/ug_partition_manager.rst

```diff
@@ -82,6 +82,113 @@ span: list
    When creating sub_partitions, this option lists what partitions the sub_partitions should span across.
    This property cannot be used together with the `placement` property.
 
+region: string
+   The memory region (address space) this partition resides in.
+   The placement of partitions is solved separately for each region.
+   The runtime and lifetime of a region is not solved separately.
+   Also, the regions share the same namespace for partition names.
+
+runtime: dict
+   When this partition is active (running).
+   Only image partitions can have a runtime.
+   The valid keywords are:
+
+      before: list
+         This partition is active before all partitions in the list.
+         This typical indicates that it boots those partitions,
+         but it doesn't indicate that the partitions in the list run concurrently (this must be indicated explicitly).
+
+      after: list
+         This partition is active after all partitions in the list.
+         This typical indicates that it is booted by one of those partitions.
+
+      alongside: list
+         This partition is active at the same time as all partitions in the list.
+         This indicates that they run concurrently, and that they must share the
+         available space in all regions.
```

Comment:
It just means they cannot overlap. When they have runtimes `alongside` each other, they are running at the same time, which means their partitions cannot overlap because they are present at the same time. This applies to all regions (they cannot overlap in FLASH or in RAM).

I'm changing the text now, putting some of it under lifetimes instead.

Reply:
done